seed/views/v3/tax_lot_properties.py

```diff
@@ -120,7 +120,7 @@ def export(self, request):
             prefetch_related = ['labels']
             filter_str = {'property__organization_id': org_id}
             if ids:
-                filter_str['property__id__in'] = ids
+                filter_str['id__in'] = ids
```

Comment:
Note that we are making the query on the *View models here, which is why this is changed to just `id`